certification/pyxis/pyxis.go

```diff
@@ -299,6 +299,10 @@ func (p *pyxisClient) GetProject(ctx context.Context) (*CertProject, error) {
 }
 
 func (p *pyxisClient) updateProject(ctx context.Context, certProject *CertProject) (*CertProject, error) {
+	// We cannot send the project type to pyxis in a Patch.
+	// Rely on omitempty to strip it from the JSON submission.
+	certProject.Type = ""
```

Comment:
This will change the project from the caller, since it's a pointer. Are we sure we want to do this?

Reply:
Don't think this is used anywhere.. except in tests.. but probably shouldn’t modify it either..

Reply:
Fair - Fixed, latest commit copies all of the data into a new struct.